### PR TITLE
Hook up keyword router to wizard

### PR DESCRIPTION
**Pull request by jerith**

### CODE REVIEW

File: go/wizard/views.py [hunk 1]
```diff
@@ -1,89 +1,149 @@
...
+        # Create channel
+        tag = self._create_channel(request, chan_form.cleaned_data)
+
+        # Create conversation
+        view_def, conversation = self._create_conversation(
+            request, conv_form.cleaned_data)
+        messages.info(request, 'Conversation created successfully.')
+
+        # Set up routing
+        logger.info("POST: %s" % (request.POST,))
```
Comment (hodgestar) [inline go/wizard/views.py]: Stray debug log?

File: go/wizard/views.py [hunk 2]
```diff
@@ -1,89 +1,149 @@
...
+        keyword = wiz_form.cleaned_data['keyword']
+        logger.info("keyword: %s" % (keyword,))
```
Comment (hodgestar) [inline go/wizard/views.py]: And here?

File: go/wizard/views.py [hunk 3]
```diff
@@ -1,89 +1,149 @@
...
+        if keyword:
+            # Create router
+            router = self._create_keyword_router(
+                request, keyword, conversation.name)
+            self._setup_keyword_routing(
+                request, conversation, tag, router, keyword)
+        else:
+            self._setup_basic_routing(request, conversation, tag)
```
Comment (hodgestar) [inline go/wizard/views.py:74]: Don't we also need to handle the case of adding a keyword to an existing keyword router?

Comment (jerith) [inline go/wizard/views.py:74]: Not for the "new channel" case. Using existing keyword routers is out of scope for this branch.

Comment (hodgestar) [inline go/wizard/views.py:74]: Do we have a relevant ticket for using existing keyword routers?

**Comment by hodgestar** [comment]
:+1: as soon as we're sure we have a ticket for using existing keyword routers.

**Comment by jerith** [comment]
I think #493 is sufficient for now.

**Comment by hodgestar** [comment]
I added a comment to #493 describing the current status.

:+1:

**Pull request merged**